src/pages/billing/Project.js

```diff
@@ -54,15 +76,18 @@ const ProjectDetail = ({ project, project: { projectName, creationStatus }, bill
   const [selectedBilling, setSelectedBilling] = useState()
   const [tab, setTab] = useState(query.tab || 'workspaces')
   const [sort, setSort] = useState({ field: 'email', direction: 'asc' })
+  const [workspaceSort, setWorkspaceSort] = useState({ field: 'name', direction: 'asc' })
 
   const signal = Utils.useCancellation()
 
   const adminCanEdit = _.filter(({ roles }) => _.includes(billingRoles.owner, roles), projectUsers).length > 1
 
   const tabToTable = {
-    workspaces: div({ style: { flexGrow: 1, width: '100%' } }, [
+    workspaces: div({ role: 'table', 'aria-label': 'workspaces list', style: { flexGrow: 1, width: '100%' } }, [
+      h(WorkspaceCardHeaders, { sort: workspaceSort, onSort: setWorkspaceSort }),
       _.flow(
         _.filter({ workspace: { namespace: projectName } }),
+        _.orderBy([`workspace.${workspaceSort.field}`], [workspaceSort.direction]),
```

Comment:
Hold off on this, I have a more complicated implementation coming in with https://github.com/DataBiosphere/terra-ui/pull/2532